routes/cloudinary-routes.js

```diff
@@ -0,0 +1,19 @@
+import cloudinary from '../services/cloudinary';
+
+function uploadAsset(req, res) {
```

Comment:
This is not a route, it is a function, move it to the services or to the actual photo-routes file